ui/src/services/k8s/api.js

```diff
@@ -183,3 +183,18 @@ export async function createEnvironment(body) {
     return { error };
   }
 }
+
+export async function getPodInNamespace(namespace, podLabel) {
+  try {
+    return await coreV1.listNamespacedPod(
+      namespace,
+      null,
+      null,
+      null,
+      null,
+      `app=${podLabel}`,
+    );
+  } catch (error) {
+    return { error };
+  }
+}
```

Comment:
Oh, that's not a good name/signature for this method IMHO (for instance, it's a `list` action, not a `get`, since you retrieve multiple objects that match a given `labelSelector`). But let's move it to a debt ticket, I think there are other methods like this one that could be improved.

Reply:
> Oh, that's not a good name/signature for this method IMHO (for instance, it's a `list` action, not a `get`, since you retrieve multiple objects that match a given `labelSelector`). But let's move it to a debt ticket, I think there are other methods like this one that could be improved.

Make sense!
Will rename the method to `queryPodInNameSpace` maybe?